recipe/patches/0002-Redis-deps-now-build-but-do-not-link.patch

```diff
@@ -1,27 +1,25 @@
-From adbd2dba70497d8f2219ce97ee8510cc3df0e4b2 Mon Sep 17 00:00:00 2001
+From 52995c7fc6299a896fcc9e6b85a17a652771b46e Mon Sep 17 00:00:00 2001
 From: Vasily Litvinov <vasilij.n.litvinov@intel.com>
-Date: Fri, 6 Nov 2020 00:55:05 +0300
-Subject: [PATCH 03/13] Redis deps now build but do not link
+Date: Fri, 3 Dec 2021 10:46:36 -0800
+Subject: [PATCH 2/6] Redis deps now build but do not link
 
-Includes update for redis-6.0.9
-
-Signed-off-by: Vasily Litvinov <vasilij.n.litvinov@intel.com>
-Signed-off-by: Gregory Shimansky <gregory.shimansky@intel.com>
+Signed-off-by: Jiajun Yao <jeromeyjj@gmail.com>
```

Comment:
I would rather you leave at least `signed off by` lines in place, adding yours to the bottom

Reply:
Oops, this is not intentional. Probably I ran the wrong command to generate this.